common/oauth.go

```diff
@@ -11,3 +11,16 @@ type OauthContext struct {
 	RefreshToken string
 	Refresher    TokenRefresher
 }
+
+// ExchangeOAuth is the API message for ExchangeOAuth context
+type ExchangeOAuth struct {
+	Code string `jsonapi:"attr,code"`
```

Comment:
This uses jsonapi annotation, which indicates it should be put under server/ module instead the common module.